client/src/components/Map/Map.js

```diff
@@ -11,20 +12,36 @@ class Map extends Component {
     },
     zoom: 16
   };
- 
+  state = {
+    lat: "",
+    lng: "",
+    center: "",
+  }
+  componentWillMount = () =>{
+    navigator.geolocation.getCurrentPosition(this.currentCoords)
+  }
+
+  currentCoords = (position) => {
+    const latitude = position.coords.latitude
+    const longitude = position.coords.longitude
+    this.setState({
+      center: {lat: latitude, lng: longitude}
+    })
+  }
+
   render() {
     return (
       // Important! Always set the container height explicitly
```

Comment:
Maybe we should do destructuring from state so we don't have to repeat `this.state...`
Something like `const { center } = this.state;`